caches/clan.go

```diff
@@ -0,0 +1,105 @@
+package caches
+
+import (
+	"fmt"
+	"math/rand"
+	"time"
+
+	gocache "github.com/patrickmn/go-cache"
+	"github.com/topfreegames/khan/models"
+)
+
+// ClansSummaries represents a cache for the RetrieveClansSummaries operation.
+type ClansSummaries struct {
+	// Cache points to an instance of gocache.Cache used as the backend cache object.
+	Cache *gocache.Cache
+
+	// The cache for one clan will live by a random amount of time within [TTL - TTLRandomError, TTL + TTLRandomError].
```

Comment:
Go comments should start with the name of the field/function/type they're about.

Eg:

// TTL is the ...

Reply:
Done!